rancher/install.go

```diff
@@ -38,12 +38,21 @@ func DeployRancherManager(hostname, channel, version, headVersion, ca, proxy str
 	if envPW := os.Getenv("RANCHER_PASSWORD"); envPW != "" {
 		password = envPW
 	}
+
 	channelName := "rancher-" + channel
+	var chartRepo string
+
+	switch channel {
+	case "prime":
+		chartRepo = "https://charts.rancher.com/server-charts/prime"
+	case "latest":
+		chartRepo = "https://releases.rancher.com/server-charts/latest"
+	case "stable":
+		chartRepo = "https://releases.rancher.com/server-charts/stable"
+	}
 
 	// Add Helm repository
-	err := kubectl.RunHelmBinaryWithCustomErr("repo", "add", channelName,
-		"https://releases.rancher.com/server-charts/"+channel,
-	)
+	err := kubectl.RunHelmBinaryWithCustomErr("repo", "add", channelName, chartRepo)
 	if err != nil {
```

Comment:
In Golang you can write like this:
```suggestion
	if err := kubectl.RunHelmBinaryWithCustomErr("repo", "add", channelName, chartRepo); err != nil {
```